src/internal/kopia/wrapper_test.go

```
@@ -483,7 +483,7 @@ func (suite *KopiaIntegrationSuite) TestBackupCollections_ReaderError() {
 		tags,
 		true,
 		fault.New(true))
-	require.NoError(t, err)
+	require.Error(t, err)
```

updated because the fault is set to fail fast